theme_bewise/views/customizations.xml

```diff
@@ -1104,4 +1120,15 @@
     </xpath>
 </template>
 
+<!-- ======== COMPANY TEAM SHAPES ======== -->
+<template id="configurator_s_company_team_shapes" inherit_id="website.configurator_s_company_team_shapes">
+    <xpath expr="//section" position="attributes">
+        <attribute name="data-oe-shape-data">{"shape":"web_editor/Origins/05",'colors':{'c3': '#FFFFFF'}}</attribute>
+    </xpath>
+    <!-- Shape -->
+    <xpath expr="//section//div" position="before">
+        <div class="o_we_shape o_web_editor_Origins_05" style="background-image: url('/web_editor/shape/web_editor%2FOrigins%2F05.svg?c3=%23FFFFFF');"/>
+    </xpath>
```

Comment:
`//section//div` seems generic, can't we use `<xpath expr="//div[hasclass('container_small')]" position="before">` instead :thinking: ?